server/init.py

```diff
@@ -18,24 +18,24 @@ def handle_exit(self, sig: int, frame) -> None:
 
 async def main():
     "Run scheduler and the API"
-    url_parser = urllib.parse.urlparse(HOSTNAME)
-    host_ip = url_parser.hostname
-    port_num = url_parser.port
-    logger.info("Conserver binding to host: {} port: {}".format(host_ip, port_num))
+    #url_parser = urllib.parse.urlparse(HOSTNAME)
```

Comment:
Why have we disabled the ability to bind to s specific IP and port?

Reply:
When we run the conserver locally, in development, we want the "hot reload" function active - when we save a new version of the source code, we want the web server to restart.  Having uvicorn inside the conserver prevents that, and as you have noted, it's not the best choice for production.